tests/runtests.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
```

Comment:
If you believe running the tests using this file with the django.setup(), then using `./manage.py test` is not required and you can instead use this file, but if you prefer the django project way using `./manage.py test` then this file is not required.